Sources/BSONCore/TopLevelDecodableValue.swift

```diff
@@ -17,14 +17,26 @@ public protocol TopLevelDecodableValue: ReadableFromByteArray {
 
     /// Reads and initializes a value from a sequence of bytes.
```

Comment:
Need more details to contrast this method from `init(bytes:)` in `ReadableFromByteArray`.
```suggestion
    /// Reads a full BSON value's encoded bytes to initialize an instance.
```